docs/rules/prefer-reflect.md

```diff
@@ -54,12 +54,12 @@ Examples of **correct** code for this rule when used without exceptions:
 ```js
 /*eslint prefer-reflect: "error"*/
 
-Reflect.apply(undefined, args);
-Reflect.apply(null, args);
+Reflect.apply(obj.foo, undefined, args);
```

Comment:
I think the above examples of incorrect patterns are badly written (sorry). Above there is `foo` (a function var) and `obj.foo` (a function property on an object). So @kaicataldo is right, this below example should demonstrate both - this should be `foo`.